README.md

```diff
@@ -2,42 +2,95 @@
 
 ## Applications
 
-Applications are located in bin directory. Before running those, please make
-sure that required Python packages (in python directory) are available:
+Applications are located in bin directory.
+Before running those, please make sure that required Python packages (in python directory) are available:
 
+```bash
 python3.8 -c "from lsst.ts.cRIOpy.GUI import *"
+```
+
+The above command shall pass without error.
+
+### M1M3GUI
+
+GUI/EUI for M1M3 static supports.
+Allows to perform various M1M3 operations, including FA bump testing and manual mirror positioning.
+Requires [ts_m1m3support](https://github.com/lsst-ts/ts_m1m3support) CSC running.
+
+To run the GUI with the CSC in simulation mode under the docker container, you can build the docker image of CSC first by:
+
+```bash
+git clone https://github.com/lsst-ts/ts_m1m3support.git
+cd ts_m1m3support
+docker build -t m1m3sim .
+```
+
+Enter the container with the display setting (use Mac OS as an example):
+
+```bash
+xhost +
+IP=$(ifconfig en0 | grep inet | awk '$1=="inet" {print $2}')
+docker run -it --rm -e DISPLAY=${IP}:0 -v ${path_to_local_ts_cRIOpy}:/home/saluser/ts_cRIOpy -v /tmp/.X11-unix:/tmp/.X11-unix m1m3sim:latest
+```
 
-shall pass without error.
+It is noted that the setting of XQuartz should allow any client: [Running GUI’s with Docker on Mac OS X](https://cntnr.io/running-guis-with-docker-on-mac-os-x-a14df6a76efc).
```

Comment:
Please note XQuartz settings should... Or just "XQuartz settings.." Still don't understand your Mac bias, when we can more easily run that on Linux desktop, but if you wish so..

Reply:
Updated in the commit: 535c3c1. Thanks!